README.md

```diff
@@ -16,13 +16,13 @@ The root folder is used for declaring parameters required by the templates, and
 |   |-- terraform/
```

Comment:
when you rebase u'll need to fix this too 
https://github.com/commitdev/zero-aws-eks-stack/blob/0dd24b6f039fe10be8f1edb333ded6857767e549/templates/terraform/bootstrap/secrets/main.tf#L35-L49